### DISCUSSION TITLE
Showcase

### DISCUSSION THREAD

**Discussion by parasyte** [parasyte/pixels#297] [Show and tell]
This discussion showcases projects using `pixels`. Feel free to add a screenshot and link to your own projects, or other apps you are aware of that use it!

# [Eldiron](https://github.com/markusmoenig/Eldiron)

Eldiron is a creator for classic RPGs written in Rust. Eldiron v1 will be able to create games similar to Ultima 4 and 5 but with a modern twist and features.

![Eldiron screenshot](https://github.com/markusmoenig/Eldiron/raw/697e18ba65980dd8c88ecd3f08afee4e2e06e9f5/docs/moody_goes_raiding_2.gif)

# [Sandbox](https://github.com/JMS55/sandbox)

A sand simulation game

![Sandbox screenshot](https://github.com/JMS55/sandbox/raw/master/flatpak/screenshot.png)

# [binocle](https://github.com/sharkdp/binocle)

_binocle_ is a graphical tool to visualize binary data. It colorizes bytes according to different rules and renders them as pixels in a rectangular grid. This allows users to identify interesting parts in large files and to reveal image-like regions.

![binocle screenshot](https://user-images.githubusercontent.com/456942/159184197-7ef43cd0-4fed-45c3-ae3a-250e1d48c120.png)

**Comment by parasyte** [comment]
# [fox32](https://github.com/fox32-arch/fox32)

**fox32** (stylized in all lowercase) is a 32 bit fantasy computer architecture, with a custom operating system and user interface inspired by various classic computers.

![fox32 screenshot](https://github.com/fox32-arch/fox32/raw/11cec386a5029322b51c3503d23d761fd25ad3cf/docs/screenshots/screenshot_fox32rom.png)


# [FunUTD](https://github.com/SamiPerttu/funutd)

FunUTD is a 3-D [procedural texture](https://en.wikipedia.org/wiki/Procedural_texture) library running on the CPU. This is an alpha version undergoing rapid development and may contain rough edges.

![FunUTD screenshot](https://github.com/SamiPerttu/funutd/raw/0f32de410b1f3a0ba24528e4661954deec4fbe1e/editor_gui.jpg)


# [NES Bundler](https://github.com/tedsteen/nes-bundler)

Transform your NES-game into a single executable targeting your favourite OS!

![nes-bundler screenshot](https://github.com/tedsteen/nes-bundler/raw/9ee2c9547403c681f5988d0ec00b8ac819431b19/screenshot.png)


# [Argentum](https://github.com/NightShade256/Argentum)

Argentum is a Game Boy Color emulator written in Rust.

![Argentum screenshot](https://github.com/NightShade256/Argentum/raw/55696e7849878741d92cfbcc173fe704686f46db/README/Zelda.png)


# [i8080](https://github.com/XAMPPRocky/i8080)

An Intel 8080 emulator in Rust

![i8080 screenshot](https://github.com/XAMPPRocky/i8080/raw/55dac142b1c9cee750c98af6785df71e52e5332b/invaders.gif)

**Comment by markusmoenig** [comment]
# [CodeEditor](https://github.com/markusmoenig/code_editor)

A code editor for pixels.
<img width="712" alt="screenshot" src="https://user-images.githubusercontent.com/10293923/187102812-c546d25a-919d-4286-98e9-f1ac4e6ac4c4.png">

**Comment by dbalsom** [comment]
# [MartyPC](https://github.com/dbalsom/martypc/)

MartyPC is a cycle-accurate IBM PC/XT emulator.

![martypc_custom_renderer_02](https://github.com/parasyte/pixels/assets/7229541/488eb50d-4a2f-4772-80e4-11efc0968d8b)

MartyPC 0.2.0 demonstrates a custom Pixels renderer that supports multiple scaling options, scaler margins (to clear egui menu bar), aspect correction, and buffer-constrained shader effects.
